src/content/tutorials/fleet-management/insfrastructure-management/crossplane/requirements/index.md

```diff
@@ -0,0 +1,50 @@
+---
+linkTitle: Crossplane requirements
+title: Crossplane requirements
+description: Resource requirements and risks Crossplane impose on the management cluster.
+weight: 20
+menu:
+  principal:
+    identifier: tutorials-fleet-management-infrastructure-crossplane-requirements
+    parent: tutorials-fleet-management-infrastructure-crossplane
+last_review_date: 2024-11-29
+user_questions:
+  - How do I configure Crossplane in the Giant Swarm platform?
+owner:
+- https://github.com/orgs/giantswarm/teams/team-honeybadger
+---
+
+## Requirements
+
+`Crossplane` significantly impacts the management cluster, depending on the `provider(s)` used.
+
+### Resource usage
+
+The providers require a significant amount of memory to run stable. For the largest, official AWS provider, we sometimes experienced 10+ GB of memory requests to run. This depends on the number of custom resource definitions (CRDs) and custom resources (CRs) managed by the provider pod.
+
+### Deploying hundreds of CRDs
+
+Managing cloud providers resources via Kubernetes CRs is one of the best thing of `Crossplane`, but also causes a lot of issues in its current state, because most cloud providers impose hundreds of new custom resource definitions to the management cluster:
+
+- official AWS provider `v0.31.0` has `894` CRDs
+- official Azure provider `v0.29.0` has `705` CRDs
+- official GCP provider `v0.29.0` has `894` CRDs
+
+This - depending on the number of custom resource definitions - impose possible [performance issues](https://github.com/crossplane/crossplane/blob/10f1e90ebb4178d4773f54bd5d7f57e7137f2d77/design/one-pager-crd-scaling.md)
+to the management cluster, including:
+
+- the API servers will take up much more memory to handle the increase in custom resource definitions (around [~3MB](https://github.com/crossplane/crossplane/issues/3754#issue-1578890243) and up to 4.5MB from Giant Swarm estimation per custom resource definition)
+- Other `Kubernetes` operators - especially ones using an older `client-go` version than `v0.25.0` - might experience throttling and timeouts towards the API. In that version, the request burst limits were increased to 300. Still, in our experience, this only partly solves the issue but needs to be more appropriately applied across the `Kubernetes` landscape. The relevant upstream issue can be found [here](https://github.com/crossplane/crossplane/issues/3272).
```

Comment:
The issue is already closed; we can remove this paragraph? or do we want to advertise the problems the deployment can cause?